### DISCUSSION TITLE
بهبود سیستم لیست هشدار

### DISCUSSION THREAD

**Discussion by dev-a-loper** [Webamoozcom/warning-list#28] [General]
لیست هشداری که در حال حاضر هستش به خودی خود خیلی چیز خوبه و خیلی پایدار تر از کانال هستش به نظر من ولی یه سری مشکلات هستش :
این که اضافه کردن یه پروژه به این لیست کار هر کسی نیست 
* یعنی باید یه دانش حداقل ابتدایی برنامه نویسی داشته باشه و بتونه با گیتهاب کار کنه 
* بعد این که باید حتما راه مستقیمی نیستش که بشه توی این صفحه گیتهاب ریپورت کرد چیزی رو 
* اسناد کافی اینجا نیست فقط یه اسمه 
* جزئیات در کل خیلی کمه
در واقع ایده ای که به ذهن من میاد اینه که وب آموز اصلش توی این مدل سایت باشه و قابل جستجو 
تا یه کانال که یه عدهی محدود میبینندش
درسته که سایت وب آموز هستش ولی خب فکر نمی کنم تمام محتوی توش قرار بگیره 

در واقع دید من اینه که کانال تبدیل بشه به یه جایی که رفرنس میده به سایت تا این که بخواد کل محتوی رو ارائه کنه ... 

شرمنده وقت نداشتم متن قشنگ تری بنویسم :pray:

**Comment by Webamoozcom** [comment]
-افزودن قسمت گزارش در صفحه لیست هشدار
-جزئیات بیشتر
-لینک به بررسی های کانال

**Comment by Webamoozcom** [comment]
بسیاری از گروهها لینک نمیشه منتشر کرد بشه لیست رو تبدیل به تصویر کرد خیلی بهتره
